src/opower/utilities/evergy.py

```diff
@@ -0,0 +1,102 @@
+"""Evergy."""
+
+from html.parser import HTMLParser
+
+import aiohttp
+from aiohttp.client_exceptions import ClientResponseError
+
+from .base import UtilityBase
+
+
+class EvergyLoginParser(HTMLParser):
+    """HTML parser to extract login verification token from Evergy Login page."""
+
+    def __init__(self, *, convert_charrefs: bool = True) -> None:
+        """Initialize."""
+        super().__init__(convert_charrefs=convert_charrefs)
```

Comment:
Looks like it is the default. Removing it does still work. I was using `pse.py` as a template since it's similar to Evergy's login and the code was in there.

https://github.com/tronikos/opower/blob/6604f311e2ca0bdb43e50fed16f48cea19ab739f/src/opower/utilities/pse.py#L12-L19

I've removed the argument and hopefully I did the argument inheritance correctly. Python isn't my native togue.